apps/api/src/persistence/database/types/ArangoEdgeDocument.ts

```diff
@@ -15,13 +16,16 @@ type ArangoEdgeMemberContext = {
     context: DTO<EdgeConnectionContext>;
 };
 
+// TODO make this extensible!
```

Comment:
Add a story for this.

Reply:
[here](https://www.pivotaltracker.com/story/show/187646934) is the story